app/services/decidim/backup_service.rb

```diff
@@ -69,29 +73,31 @@ def create_backup_export
 
     def backup_database
       if can_connect_to_db?
-        file = generate_backup_file_path("db", "dump")
+        file_path = generate_backup_file_path("db", "dump")
 
         cmd = "pg_dump -Fc"
         cmd += " -h '#{@options[:db_conf][:host]}'" if @options[:db_conf][:host].present?
         cmd += " -p '#{@options[:db_conf][:port]}'" if @options[:db_conf][:port].present?
         cmd += " -U '#{@options[:db_conf][:username]}'" if @options[:db_conf][:username].present?
         cmd = "PGPASSWORD=#{@options[:db_conf][:password]} #{cmd}" if @options[:db_conf][:password].present?
         cmd += " -d '#{@options[:db_conf][:database]}'" if @options[:db_conf][:database].present?
-        cmd += " -f '#{file}'"
+        cmd += " -f '#{file_path}'"
 
         Rails.logger.info("Started backup_database with #{cmd}")
 
-        execute_backup_command(file, cmd)
+        execute_backup_command(file_path, cmd)
       else
         Rails.logger.error "Cannot connect to DB with configuration"
         Rails.logger.error @options[:db_conf].except(:password)
         Rails.logger.error "DB password was #{@options[:db_conf][:password].present? ? "present" : "missing"}"
         # do not exit here because we can still try to do the other backups
+        false
       end
     end
 
     def backup_uploads
-      if File.exist?("storage")
+      # TODO: What is storage in this context?
+      if file_exists?("storage")
```

Comment:
What is storage in this context?

Reply:
"storage" is the path where Active Storage stores files in local mode.
I didn't found an easy way to get that configuration entry from Active Storage so it's hard coded for now.
Anyway, the BackupService won't be used in k8s production because we will backup things from source (buckets, env vars and database)